docs/release-notes.txt

```diff
@@ -8,6 +8,17 @@
  Version 0.15
 **************
 
+Version 0.15.3
+==============
+
+**Release Date:** May 5, 2021
+
+**Bug Fixes**
+
+-  Images: Fix GPU support in CUDA 10.2 + Tensorflow 1.15 images.
+-  Trials: Fix trial environment dependency compatibility for
```

Comment:
non-blocking: Seems a bit too implementation-detail focused (also afaik this wasn't our fault); maybe "Update to match `websockets` library API change." or something? @mackrorysd maybe I have this wrong

Reply:
@mackrorysd Any comment on this or should I just ship it?

Reply:
I wrote that line. I'm fine with either version as long as it mentions `websockets>=9.0`.

Reply:
Sorry I must have missed the earlier tag. Maybe "Update to match websocket >= 9.0 library API change" to be specific and emphasize it's an incompatibility in a new version of a dependency.